test_launch_ros/test/test_launch_ros/actions/test_node.py

```diff
@@ -57,7 +57,7 @@ def _assert_type_error_creating_node(self, *, parameters=None, remappings=None):
     def test_launch_invalid_node(self):
         """Test launching an invalid node."""
         node_action = launch_ros.actions.Node(
-            package='nonexistent_package', node_executable='node', output='screen'),
+            package='nonexistent_package', node_executable='node', output='screen')
```

Comment:
We think this was always causing a subtle bug (`node_action` was a single item tuple rather than just a single item), but was being hidden because the tests didn't complete due to the race condition being fixed in this pull request.